application/src/main/java/com/musicforall/model/Track.java

```diff
@@ -102,7 +118,7 @@ public Integer getId() {
         return id;
     }
 
-    public void setId(Integer id) {
+    private void setId(Integer id) {
```

Comment:
As far as I know, the accessor methods are required for a class to comply with JavaBean standard. And the setId method is private by design, since it should never be called explicitly.